.github/workflows/send-screenshots-to-pixeleagle.yml

```diff
@@ -16,18 +16,29 @@ on:
         required: true
         type: string
 
+env:
+  # Unfortunately, we can't check secrets in `if:` conditionals. However, Github's own documentation
+  # suggests a workaround: Putting the secret in an environment variable, and checking that instead.
+  PIXELEAGLE_TOKEN_EXISTS: ${{ secrets.PIXELEAGLE_TOKEN != '' }}
+
 jobs:
   send-to-pixel-eagle:
     name: Send screenshots to Pixel Eagle
     runs-on: ubuntu-24.04
     steps:
+      - name: Notify user on non-existent token
+        if: ${{ env.PIXELEAGLE_TOKEN_EXISTS == 'false' }}
```

Comment:
In case you're thinking "why is this a comparison to a stringified boolean?": environment variables are strings, even those that are booleans.

It took me way too long to figure this out.